README.md

```diff
@@ -1,86 +0,0 @@
-## Quick start
```

Comment:
I don't understand what happened that caused the README.md to disappear here.

Copying from slack:

> So, as usual when thinking "who did this?", it was me who moved the README.md in b4bf835dc102d70ea889c9ed5c279b6cee6080d0 . Luckily, you had my back and you added a README.md back in in cec671ae479c3d29957397516b8e56f9320a8ebd .
> 
> But for some reason it is not in the mb/multiinputs branch
> 
> I don't particularly care about the README, since we can easily put that back, but I think we should know what happened to it, because maybe there are other things missing that would be important
>